src/main/java/org/traccar/WebDataHandler.java

```diff
@@ -54,15 +57,38 @@
     private final String header;
     private final boolean json;
 
+    private final Boolean retryEnabled;
+    private final Integer retryDelayMin;
+    private final Integer retryDelayMax;
+    private final Integer deliveryPendingLimit;
+
+    private AtomicInteger deliveryFaliuresInRow;
+    private AtomicInteger deliveryPendingCurrent;
+
     @Inject
     public WebDataHandler(
             Config config, IdentityManager identityManager, ObjectMapper objectMapper, Client client) {
+
         this.identityManager = identityManager;
         this.objectMapper = objectMapper;
         this.client = client;
         this.url = config.getString(Keys.FORWARD_URL);
         this.header = config.getString(Keys.FORWARD_HEADER);
         this.json = config.getBoolean(Keys.FORWARD_JSON);
+
+        this.retryEnabled = config.getBoolean(Keys.FORWARD_RETRY_ENABLE);
+
+        Integer retryDelayMin = config.getInteger(Keys.FORWARD_RETRY_DELAY_MIN, 1);
```

Comment:
After removing validations this will simplify.

Reply:
Ok.